contribute/CONTRIBUTING-TO-DEVELOPER-DOC.md

```diff
@@ -32,8 +33,7 @@ You can also help with some existing issues under this category available at [de
 - Fork the openebs repository and if you had previously forked, rebase with master to fetch latest changes
```

Comment:
OpenEBS repository and if you have already forked it, rebase with the master...changes.

Reply:
Changed it to, 
- Fork the [OpenEBS](www.github.com/openebs/openebs) repository and if you have forked it already, rebase with master branch to fetch latest changes to your local system.